servicecontrol/servicecontrol-in-practice.md

```diff
@@ -1,35 +1,40 @@
+- For each endpoint, turn on Auditing (if needed) and re-run the performance tests to determine impact.
+- For each endpoint, turn on any required Custom Checks and re-run the performance tests to determine impact.
 
-- If running anti-virus software exclude the ServiceControl [database directory](/servicecontrol/configure-ravendb-location.md) from virus checks.
-- The recommended approach is that customers don't downgrade major and minor releases of ServiceControl. ServiceControl uses an embedded database and changes to the internal data structures can occur between releases. Rolling back may cause index corruption or data loss. The recommended approach is that customers perform testing in a lower environment before upgrading in production environments.
\ No newline at end of file
+NOTE: There are some factors that influence ServiceControl resource usage and performance that are less likely to change between environments but are specific to the system being monitored such as choice of transport and average message size. 
+
+
+## Specific Considerations
+
+- Each environment should have a dedicated ServiceControl instance.
+- In production, ServiceControl should run on a **dedicated machine**.
+- Turn off [Message Auditing](/nservicebus/operations/auditing.md) if it is not needed. The audit ingestion capability of ServiceControl is primarily to support system analysis with ServiceInsight. If ServiceInsight is not deployed in an environment then configure ServiceControl to read audit messages from an empty queue or turn off messaging auditing for each endpoint. Message auditing may be important for some endpoints but not other others. 
+- Turn off [Audit Forwarding](/servicecontrol/errorlog-auditlog-behavior.md) if it is not needed. ServiceControl sends a copy of each audited message to configured Audit Forwarding queue. If these messages are not being used, turn this feature off.
+- Plugins communicate between the Endpoint they installed in to ServiceControl using messaging and the configured transport of your system. Each instance of a plugin adds more messages to the ServiceControl queue which can delay the processing of each message and make ServiceControl less responsive.
+  - [Heartbeats](/servicepulse/intro-endpoints-heartbeats.md) - Not all endpoints are mission critical and need to be monitored with heartbeats. For Endpoints that do need to be monitored, adjust the Heartbeat Interval. Increasing the interval ensures that ServiceControl is able to process heartbeats in a timely manner but also increases the window that an Endpoint can be unavailable for before being detected by ServiceControl. Heartbeat messages tend to be frequent and a large backlog can occur if ServiceControl is offline for an extended period. When this happens, it can take ServiceControl some time to process old heartbeats when it restarts. 
```

Comment:
What does make one endpoint mission critical, and others not?

Isn't it about how fast this endpoint needs to process the messages and if it's a big deal if you had say 10 min. delay?
Would be good also to ensure the user here that even though the endpoint is down, the messages won't be lost. So once it goes up again it'll pick them up. It's not a loss, just delay. Not "what messages can you lose (because the endpoint is not mission critical)", but "which you can process with delay without losing your job".

I think it's important to make that clear here.